launcher/launcher/main.go

```diff
@@ -155,6 +157,28 @@ func startLauncher(ctx context.Context, launchSpec spec.LaunchSpec, serialConsol
 	}
 	defer containerdClient.Close()
 
+	if launchSpec.Experiments.EnableGpuDriverInstallation && launchSpec.InstallGpuDriver {
+		gpuType, err := deviceinfo.GetGPUTypeInfo()
+		if err != nil {
+			return fmt.Errorf("failed to get the gpu type info: %v", err)
+		}
+
+		if !gpuType.OpenSupported() {
+			return fmt.Errorf("open-sourced gpu kernel modules are not supported for the gpu type %s", gpuType.String())
```

Comment:
maybe list all supported GPUs in the error mesg

Reply:
done